src/cts/src/TreeBuilder.cpp

```diff
@@ -73,6 +73,44 @@ void TreeBuilder::initBlockages()
                   "{} placed hard macros will be treated like blockages.",
                   bboxList_.size());
   }
+
+  // add tree buffer width and height for legalization
+  std::string buffer;
+  if (!options_->getTreeBuffer().empty()) {
+    buffer = options_->getTreeBuffer();
+  } else {
+    buffer = options_->getRootBuffer();
+  }
+  odb::dbMaster* libCell = db_->findMaster(buffer.c_str());
+  assert(libCell != nullptr);
+  bufferWidth_ = (double) libCell->getWidth() / techChar_->getLengthUnit();
+  bufferHeight_ = (double) libCell->getHeight() / techChar_->getLengthUnit();
+  // clang-format off
+  debugPrint(logger_, CTS, "legalizer", 3, "buf width= {:0.3f} buf ht= {:0.3f} scalingUnit={}",
+             bufferWidth_, bufferHeight_, techChar_->getLengthUnit());
+  // clang-format on
```

Comment:
Remove clang-format off/on comments. Even if the clang-format uses more lines for cases like this, I believe it's more readable.

Reply:
I respectfully disagree. I don't want to see 10 lines for a debug statement.   This creates too much clutter and degrades readability in my opinion.

Reply:
I don't have a strong opinion on the number of lines and strictly following clang-format, but we should avoid lines with more than 80 columns. It's okay for me if you tackle this in a future PR, but we should avoid adding more lines like that.

Reply:
Agree.  I fixed all long debugPrint lines in TreeBuilder.cpp.